cmd/access/node_builder/access_node_builder.go

```diff
@@ -347,6 +346,7 @@ func (builder *FlowAccessNodeBuilder) buildFollowerEngine() *FlowAccessNodeBuild
 			node.Storage.Headers,
 			builder.Finalized,
 			core,
+			followereng.WithComplianceConfigOpt(modulecompliance.WithSkipNewProposalsThreshold(node.ComplianceConfig.SkipNewProposalsThreshold)),
```

Comment:
It doesn't make a huge difference IMO whether we use height or view here, but:
- The Follower uses view
- The Participant uses height
- The documentation mentions height
https://github.com/onflow/flow-go/blob/7167ee3f1bd803aa460f64e28d5fabc52fc26335/module/compliance/config.go#L8-L12

### Suggestion

The Participant engine has easy access to the finalized view, and the view of the header being considered.
https://github.com/onflow/flow-go/blob/5346e7b437b6fa72497b3a16ea7cb5d426f1556d/engine/collection/compliance/core.go#L121

We can:
- change the documentation for `SkipNewProposalsThreshold` to refer to view rather than height
- change [this line](https://github.com/onflow/flow-go/blob/5346e7b437b6fa72497b3a16ea7cb5d426f1556d/engine/collection/compliance/core.go#L153) in Participant engine to use finalized view instead of height

Then the two implementations, and config documentation, will all be consistent.

Reply:
Agreed, the name doesn't refer to view or height - but the documentation for the config (used in both implementations) explicitly refers to height. 
> Formally, let `H` be the **height** of the latest finalized block known to this node. A new block `B` is dropped without further processing, if B.**Height** > H + SkipNewProposalsThreshold

Reply:
+1 on Jordan's suggestion to make it consistent. I have a mild inclination to using view, because that is the canonical measure for time in our protocol. For example, if we use view, we could in the future eliminate an additional edge case in the follower: https://github.com/onflow/flow-go/blob/736d2326f8a71db76bbc8e9d3a1ac2591b23c9d7/engine/common/follower/compliance_core.go#L157-L159